main/bl_main.c

```diff
@@ -239,12 +239,12 @@ uint32 getProfilerTimerCount() {
 
 void vApplicationStackOverflowHook(TaskHandle_t xTask, char *pcTaskName) {

```

Comment:
I didn't know about this hook - can we use it to print a log message or return address for user code?

Reply:
It would be useless to print here in the bootloader, but we can try to log it in the OBC software. I don't know if it will be able to write to the SD card since this function runs in kernel mode

Reply:
I don't know if we can do core dumps from space, but it would be nice. I was thinking it could append to the in-memory log or write to a post mortem area, kind of like the register dump we'd like to do after a SEGV.